src/js/App.js

```diff
@@ -0,0 +1,10 @@
+import Modal from "./modal/Modal.js";
+import MainPage from "./main/MainPage.js";
+import { $, $$ } from "./utils.js";
+
+export default class App {
+    constructor(){
+        this.Modal = new Modal()
+        this.MainPage = new MainPage()
+    }
```

Comment:
```suggestion
    Modal, MainPage;
    constructor(){
        this.Modal = new Modal()
        this.MainPage = new MainPage()
    }
```
이렇게 클래스의 프로퍼티 영역에 어떤 변수를 사용할건지 미리 선언해주면 다른 사람이 코드를 볼 때 이해하기가 더 수월하답니다!

Reply:
오 저런 식으로 적어도 코드 상에서는 별다른 오류가 없나보네요! 혹시 이런 선언 방식을 어떻게 부르면 될까요? 멤버변수로 이해해도 괜찮을까요?

Reply:
`멤버변수 선언` 이라고 생각하시면 된답니다!